go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/lightningnetwork/lnd/queue v1.1.1
 	github.com/lightningnetwork/lnd/sqldb v1.0.3
 	github.com/lightningnetwork/lnd/ticker v1.1.1
-	github.com/lightningnetwork/lnd/tlv v1.2.3
+	github.com/lightningnetwork/lnd/tlv v1.2.6
```

Comment:
You mean pull it out into a separate PR? I guess I could extract some independent commits into its own PR and get that merged sooner. But the initial plan was to get everything in as a whole...

Reply:
I'm fine with leaving it where it is. I'm mostly saying this because I know I've had trouble getting large PRs in and things go faster when smaller chunks are made. If you wanna leave it where it is go for it.

Reply:
Ended up extracting it into its own PR.